cpp/pybind/t/geometry/image.cpp

```diff
@@ -101,6 +101,22 @@ void pybind_image(py::module &m) {
                  "tensor"_a);
     docstring::ClassMethodDocInject(m, "Image", "__init__",
                                     map_shared_argument_docstrings);
+
+    // Pickle support.
+    image.def(py::pickle(
+            [](const Image &image) {
+                // __getstate__
+                return py::make_tuple(image.AsTensor());
```

Comment:
Do we need to transfer to CPU?

Reply:
The conversion to CPU is performed in `Tensor` side, so we don't need here.